GHzDACs/Cleanup/dac.py

```diff
@@ -677,8 +677,25 @@ def _setPolarity(self, chan, invert):
 
     @inlineCallbacks
     def _checkPHOF(self, op, fifoReadings, counterValue):
-        # Determine for which PHOFs (FIFO offsets) the FIFO counter equals
-        # counterValue.
+        """ Determine correct PHOF for FIFO counter of counterValue
```

Comment:
Point taken on the period. However, for the space, in the pyle contributing file we have a space between the `"""` and the start of the text. What do I do?